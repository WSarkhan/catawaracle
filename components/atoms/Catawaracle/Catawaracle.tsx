import React from 'react';
import { Text, View } from 'react-native';
import { CatFace } from '@/components/foundations';

export const Catawaracle = () => (
  <View className="flex flex-row pt-5">
    <Text className="text-3xl font-bold text-orange-500 ">Cata</Text>
    <View className="transform -rotate-180 ">
      <CatFace color={'rgb(251 146 60)'} />
    </View>
    <Text className="mr-2 text-3xl font-bold text-orange-500 ">aracle</Text>
  </View>
);
