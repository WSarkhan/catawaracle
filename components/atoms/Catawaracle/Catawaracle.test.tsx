import * as React from 'react';
import * as renderer from 'react-test-renderer';
import { Catawaracle } from './Catawaracle';

describe('Catawaracle', () => {
  it('should match a snapshot', () => {
    const tree = renderer.create(<Catawaracle />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
