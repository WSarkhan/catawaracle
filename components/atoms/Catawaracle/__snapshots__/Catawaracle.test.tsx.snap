// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Catawaracle should match a snapshot 1`] = `
<View
  className="flex flex-row py-5"
>
  <Text
    className="text-3xl font-bold text-orange-500 "
  >
    Cata
  </Text>
  <View
    className="transform -rotate-180 "
  >
    <RNSVGSvgView
      align="xMidYMid"
      bbHeight="34"
      bbWidth="34"
      focusable={false}
      height="34"
      meetOrSlice={0}
      minX={0}
      minY={0}
      style={
        [
          {
            "backgroundColor": "transparent",
            "borderWidth": 0,
          },
          {
            "flex": 0,
            "height": 34,
            "width": 34,
          },
        ]
      }
      vbHeight={90}
      vbWidth={100}
      width="34"
    >
      <RNSVGGroup
        fill={
          {
            "payload": 4278190080,
            "type": 0,
          }
        }
      >
        <RNSVGGroup
          fill={
            {
              "payload": 4278190080,
              "type": 0,
            }
          }
        >
          <RNSVGPath
            d="m13.07 54.461c4.1211 0.019531 3.8906-7.3516 4.25-12.148 0.32813-4.4883 0.76172-8.9492 1.4492-13.301 0.69922-4.3281 1.6289-8.6289 3.2109-12.301 0.39062-0.92188 0.82812-1.7695 1.3203-2.5781 0.23828-0.39844 0.51172-0.80078 0.76953-1.1602 0.10938-0.16016 0.25-0.28906 0.39844-0.41016 0.28906-0.23828 0.67969-0.37891 1.0703-0.39844 0.19141 0 0.39063 0 0.57813 0.050782 0.21094 0.058594 0.28125 0.070312 0.80859 0.35156 3.5781 1.9102 6.9883 4.5781 9.4609 7.6602l0.050781 0.058594c0.98047 1.2109 2.6719 1.6914 4.1797 1.0586 3.3711-1.4219 7.0195-2.1797 10.672-2.2109 3.8008-0.03125 7.6094 0.71094 11.129 2.1602 1.4414 0.58984 3.1484 0.19141 4.1602-1.0898v-0.019531c1.3086-1.6289 2.8281-3.1484 4.5195-4.5117 0.85156-0.67969 1.7305-1.3203 2.6406-1.9219 0.89062-0.58984 1.8789-1.1797 2.7305-1.6016 0.82031-0.41016 1.8984-0.23828 2.5195 0.42969 0.10156 0.089844 0.12109 0.12891 0.26172 0.30078l0.42969 0.60938c0.26953 0.42188 0.53906 0.83984 0.78906 1.2891 0.5 0.87891 0.94141 1.8398 1.3398 2.8281 1.5898 4.0117 2.5195 8.5703 3.2109 13.16 0.67969 4.6016 1.0898 9.3008 1.3711 14.02 0.21094 3.7695-0.55859 9.5781 3.1289 9.6719 3.9688 0.10156 2.6484-5.8281 2.4883-9.5508-0.21875-4.8789-0.58984-9.7617-1.2617-14.648-0.69141-4.8789-1.6289-9.7812-3.4883-14.59-0.46875-1.1992-1.0117-2.3906-1.6484-3.5508-0.32031-0.57813-0.66016-1.1484-1.0312-1.7109-0.39062-0.55859-0.69922-1.0586-1.3086-1.7305-2.2695-2.5-6.1914-3.25-9.2109-1.7617-1.2891 0.62891-2.3086 1.2695-3.3984 1.9805-1.0703 0.69922-2.1016 1.4609-3.1016 2.2617-1.5781 1.2812-3.0703 2.6914-4.4297 4.25-3.7109-1.3281-7.6602-2.0195-11.609-2.0312-3.9297 0-7.8516 0.66016-11.559 1.9492-2.8516-3.3008-6.2305-5.9688-9.9688-8.1406-0.28125-0.16016-0.51953-0.30078-0.87109-0.46875-0.35938-0.17969-0.73828-0.32812-1.1211-0.44922-0.76953-0.23828-1.5781-0.37891-2.3984-0.39062-1.6289-0.03125-3.2617 0.44922-4.6406 1.3398-0.69141 0.44141-1.3086 1-1.8398 1.6289-0.30078 0.33984-0.42188 0.53906-0.62891 0.80859-0.19922 0.25-0.37109 0.51953-0.55078 0.78125-0.73047 1.0586-1.3398 2.1602-1.8906 3.2812-2.1719 4.4883-3.3516 9.1094-4.3516 13.699-0.98047 4.6016-1.6797 9.1992-2.2812 13.801-0.37109 2.6992-0.69141 6.0586-0.32031 8.6602 0.37891 2.6094 1.4609 4.5508 3.9609 4.5586z"
            fill={
              {
                "payload": 4294677052,
                "type": 0,
              }
            }
            propList={
              [
                "fill",
              ]
            }
          />
          <RNSVGPath
            d="m33.09 44.891c-0.71094 6.9688 10.609 6.0703 8.6289-0.32031-0.85938-4.9883-8.2891-4.8008-8.6289 0.32031z"
            fill={
              {
                "payload": 4294677052,
                "type": 0,
              }
            }
            propList={
              [
                "fill",
              ]
            }
          />
          <RNSVGPath
            d="m61.891 44.891c-0.71094 6.9688 10.609 6.0703 8.6289-0.32031-0.85938-4.9883-8.2891-4.8008-8.6289 0.32031z"
            fill={
              {
                "payload": 4294677052,
                "type": 0,
              }
            }
            propList={
              [
                "fill",
              ]
            }
          />
        </RNSVGGroup>
      </RNSVGGroup>
    </RNSVGSvgView>
  </View>
  <Text
    className="mr-2 text-3xl font-bold text-orange-500 "
  >
    aracle
  </Text>
</View>
`;
